app/scripts/components/common/element-interactive.js

```diff
@@ -109,6 +118,8 @@ export const ElementInteractive = React.forwardRef(
 );
 
 ElementInteractive.propTypes = {
+  linkTo: T.string.isRequired,
+  onLinkClick: T.func,
```

Comment:
Is there a reason you decided to add this prop now? I see `getLinkProps` already accepted an `onClick` param but wondering if you had a specific reason?

Reply:
ah, that was because of how I was passing the props for SmartLink before. But your comment made me think that I don't have to do that! Thanks!